pydoc-markdown/src/pydoc_markdown/contrib/renderers/markdown.py

```diff
@@ -205,7 +210,11 @@ def _render_toc(self, fp, level, obj):
     if level > self.toc_maxdepth:
       return
     object_id = self._resolver.generate_object_id(obj)
-    fp.write('  ' * level + '* [{}](#{})\n'.format(self._escape(obj.name), object_id))
+    title = self._escape(obj.name)
+    if not self.add_module_prefix and isinstance(obj, docspec.Module):
+      object_id = ".".join(object_id.split('.')[1:])
```

Comment:
The "object id" should not be affected by this new option. It's supposed to link to the anchor tags inserted by `insert_header_anchors`.

Reply:
ok I think that what breaks the title (module) links in TOC!

Reply:
That's why you need to enable the `markup.goldmark.render.unsafe` option in Hugo, right? Per https://github.com/NiklasRosenstein/pydoc-markdown/issues/198#issuecomment-894913526

Reply:
> That's why you need to enable the `markup.goldmark.renderer.unsafe` option in Hugo, right? Per [#198 (comment)](https://github.com/NiklasRosenstein/pydoc-markdown/issues/198#issuecomment-894913526)

No, I'm referring to the new option I'm trying to add. When using it the anchor link of the modules (mtb_blender.console in the screenshot) is not working, members are. I think it's related to how I affect also the `object_id`, I'm testing now

Reply:
Yes, that's what I was trying to point out in my first comment. 😄 The object ID is what tells the browser where to go to when you click the link. You shouldn't touch it here.

Reply:
Should be solved by 1c405f46766ca21be998741b58ea514eda9c3418